tests/conditional_compilation/test_utils.py

```diff
@@ -72,15 +72,18 @@ def run_infer(model, out_file, install_dir):
     return return_code, output
 
 
-def make_build(openvino_root_dir, build_dir, install_dir, cmake_additional_args=None, log=None):
+def make_build(openvino_root_dir, build_dir, install_dir, build_target=None, cmake_additional_args=None, log=None):
     """Parametrized build and install OpenVINO package."""
     additional_args_line = " ".join(cmake_additional_args) + " " if cmake_additional_args else ""
+    build_target_arg_line = f"cmake --build {build_dir} --target " + " ".join(
+        build_target) + " && " if build_target else ""
```

Comment:
Please update this cmd and using only string formatting  instead combine concatenate and formatting